packages/rpc-graphql/README.md

```diff
@@ -975,54 +947,38 @@ const source = `
-                                                                    }
-                                                                }
-                                                                owner {
+                                                        address
+                                                        mint {
+                                                            ... on MintAccount {
+                                                                data {
                                                                     address
-                                                                    lamports
+                                                                    decimals
                                                                 }
                                                             }
                                                         }
+                                                        owner {
+                                                            address
+                                                            lamports
+                                                        }
                                                     }
                                                 }
                                             }
                                             source {
                                                 ... on TokenAccount {
                                                     data {
-                                                        parsed {
-                                                            info {
-                                                                address
-                                                                mint {
-                                                                    ... on MintAccount {
-                                                                        data {
-                                                                            parsed {
-                                                                                info {
-                                                                                    address
-                                                                                    decimals
-                                                                                }
-                                                                            }
-                                                                        }
-                                                                    }
-                                                                }
-                                                                owner {
+                                                        address
+                                                        mint {
+                                                            ... on MintAccount {
```

Comment:
Isn't this _always_ of the `MintAccount` type? Shouldn't be any need to type the subselection.

Reply:
Yes it's always a mint account, and I could stick the `MintAccount` type in there (which would be pretty cool), but that eliminates the potential for someone to get the `mint` account as `base58`, etc. It would always be the parsed `MintAccount`.

I'm leaning towards it though, because having to specify the spread here stinks.

Reply:
One way we can do that is to let fields take arguments:

```graphql
source {
  mint(encoding: BASE_64) {
    data # Base64String
  }
}
```

Reply:
My idea [here](https://github.com/solana-labs/solana-web3.js/pull/1837#discussion_r1407188113) is better.